datasets/README.md

```diff
@@ -0,0 +1,11 @@
+MICROSOFT PROVIDES THE DATASETS ON AN "AS IS" BASIS. MICROSOFT MAKES NO WARRANTIES, EXPRESS OR IMPLIED, GUARANTEES OR CONDITIONS WITH RESPECT TO YOUR USE OF THE DATASETS. TO THE EXTENT PERMITTED UNDER YOUR LOCAL LAW, MICROSOFT DISCLAIMS ALL LIABILITY FOR ANY DAMAGES OR LOSSES, INLCUDING DIRECT, CONSEQUENTIAL, SPECIAL, INDIRECT, INCIDENTAL OR PUNITIVE, RESULTING FROM YOUR USE OF THE DATASETS.
```

Comment:
I think it would be good to put all the datasets in a single place - `test/data` or similar.  That way they aren't duplicated unintentionally, and there is one place to see all of the datasets in the repo.

Reply:
yes, was thinking about it too. I suggest moving datasets from test to the folder datasets in the root and merge them with data that is there for samples. This way it will be obvious for users where to look for it. I can do it if no objections.

Reply:
There's another competing concern of having too many "things" in the root folder.

A structure I could imagine working would be:

```
build \

docs \

samples \
  datasets \  (or just 'data'?)
  GettingStarted \
  GitHubLabeler \

src \
  Project1 \
  Project2 \

test \
  Test1 \
  Test2 \

```

And all the tests changing to instead of read from `test\data`, they now point to `samples\datasets` (or `samples\data`).